README.md

```diff
@@ -1,17 +1,12 @@
 # Vertex Web Monorepo
 
 Welcome to the Vertex Web repo. This is a monorepo that contains our front-end
-projects at Vertex.
+tooling at Vertex.
 
 ## Structure
 
 - `./packages`: The packages to be made available on NPM. This directory contains
   subdirectories that represent different categories of packages:
-  - `./libs`: The libs directory contains libraries that are intended to be
-    reusable across other applications and/or libraries. These are normally
-    published to our private NPM repository.
-  - `./sdks`: The sdks directory contains NPM packages for Vertex's public facing
-    SDKs.
   - `./tools`: The tools directory contains tooling libraries. Examples of these
```

Comment:
Maybe a follow up PR to move the packages in `./packages/tools` up a level to `./packages`.